### DISCUSSION TITLE
Storing Actions

### DISCUSSION THREAD

**Discussion by rubend056** [njaard/sonnerie#25] [Ideas]
How would one go about storing actions (which don't really have a value), just a label + timestamp. Plus aggregating those values over time.

Is this easily doable, or would you need something on top to aggregate and cache.

I'm looking to answer "How many edits were made last year/month/day".

I'm also assuming the label to be something like `created_<user>`, would that be more, or less efficient, than `created <time> s <user>`.

**Comment by njaard** [comment]
I would do `created_<user>` if you want to answer the question "how many edits did each user make", and `created <time> s <user>` if you want to answer "how many edits were made by anyone. That's because a lookup by key is very efficient.

Sonnerie doesn't do any aggregation, because there's no query language or anything. Every time you add a value, you'd have to update your aggregates, maybe that could be another series called "created_1y", "created_1m" and "created_1d".
